### ISSUE TITLE
viewport setting in metadata is not working correctly

### ISSUE THREAD

**Issue by hi-paul** [vercel/next.js#63592]
### Link to the code that reproduces this issue

https://codesandbox.io/p/devbox/priceless-rgb-2hcyms?file=%2Fapp%2Flayout.tsx&workspaceId=12540e31-6959-44ce-a050-2ec2eabe3bda

### To Reproduce

Add code to root layout.tsx

```typescript
export const metadata = {
  viewport: { width: 1920 }
}
```

### Current vs. Expected behavior

current: `<meta name="viewport" content="width=device-width, initial-scale=1" />`

expected: `<meta name="viewport" content="width=1920" />`

### Provide environment information

```bash
Operating System:
  Platform: darwin
  Arch: arm64
  Version: Darwin Kernel Version 23.2.0: Wed Nov 15 21:53:18 PST 2023; root:xnu-10002.61.3~2/RELEASE_ARM64_T6000
Binaries:
  Node: 20.11.0
  npm: 10.2.4
  Yarn: 1.22.19
  pnpm: 8.14.1
Relevant Packages:
  next: 13.5.6
  eslint-config-next: 13.5.6
  react: 18.2.0
  react-dom: 18.2.0
  typescript: 4.9.5
Next.js Config:
  output: N/A
```

### Which area(s) are affected? (Select all that apply)

App Router, Metadata (metadata, generateMetadata, next/head)

### Which stage(s) are affected? (Select all that apply)

next dev (local), next build (local), next start (local), Vercel (Deployed), Other (Deployed)

**Comment by ttaerrim** [comment]
try to add viewport code to root layout.tsx

```typescript
export const viewport: Viewport = {
  width: 1920
}
```

**Comment by hi-paul** [comment]
I recently upgraded vercel version and didn't realize that the viewport for metadata was deprecated. Thanks for letting me know🙏 @ttaerrim

When I applied it as you suggested, it looks like `<meta name="viewport" content="width=1920, initial-scale=1">`, but can't I remove the `,initial-scale=1`?

**Comment by ttaerrim** [comment]
Sorry, I deleted my comment by mistake.
> try to add code to root layout.tsx
> ```typescript
> export const viewport: Viewport = {
>   width: 1920
> }
> ```


But I don't know how to initialize viewport tag to none.

**Comment by devjiwonchoi** [comment]
Hi, `initial-scale` set to 1 will not harm since it is [the default value](https://developer.mozilla.org/en-US/docs/Web/HTML/Viewport_meta_tag#initial-scale).
Since the default value is mapped internally, you cannot explicitly remove it from your source.

https://github.com/vercel/next.js/blob/65d699e7d7b259af948990aa8052ffecc33812f5/packages/next/src/lib/metadata/default-metadata.tsx#L6-L15

Also, it is recommended to follow the Codemods so that you don't need to explicitly export `viewport`.

ref: https://nextjs.org/docs/app/building-your-application/upgrading/codemods#use-viewport-export

**Comment by hi-paul** [comment]
Actually, what I'm working on now is to show the desktop in 1920 width on the mobile screen. Before upgrading nextjs, it was `<meta name="viewport" content="width=1920" />` and it worked as I wanted, but now it doesn't work when it's set to `<meta name="viewport content="width=1920, initial-scale=1" />`.

The default value of initial-scale is 1, but I figured that its presence had an impact, so I set initial-scale to -1 and it worked fine.

Thank you for your kind explanation and help in resolving the issue.
